keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt

```diff
@@ -70,7 +70,7 @@ class ImageService(
   suspend fun getLatestNamedImage(packageName: String, account: String, region: String? = null): NamedImage? =
     cloudDriverService.namedImages(DEFAULT_SERVICE_ACCOUNT, packageName, account, region)
       .sortedWith(NamedImageComparator)
-      .lastOrNull {
```

Comment:
I changed the NamedImageComparator (and also the DebianComparator) to default to descending order. We always want the "latest", and it seemed easier to always have that first instead of finding the last of a list, or reversing the list. All the docker comparators are also sorted in descending order.

Reply:
Kotlin adds a `reversed()` function to `Comparator` so you could always do ```.sortedWith(NamedImageComparator.reversed())```

Reply:
It was really hard for me to keep track of that mentally - especially with apis that return descending order by default (like the rocket versions api). I kept getting confused - when do we have to reverse? What's the default sort order here? So I thought this might be a little bit easier.